app/src/Command.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+require_once(__DIR__ . '/Db.php');
+require_once(__DIR__ . '/XmlParser.php');
+
+class Command 
```

Comment:
What do you thing about a more explicit name, like : 
```suggestion
class SyncCommand 
```

(in case you add more commands in the future).